db/migrate/20211109185240_create_monsters.rb

```diff
@@ -0,0 +1,14 @@
+class CreateMonsters < ActiveRecord::Migration[6.1]
+  def change
+    create_table :monsters do |t|
+      t.string :name, null: false
+      t.string :habitat
+      t.text :description, null: false
+      t.string :image, null: false
+      t.string :classification
+      t.integer :votes, default: 0
```

Comment:
Let's remove votes until we're ready to work on and implement that feature (same like the image, that I see you've removed in a new migration below)